src/utils/frame.js

```diff
@@ -160,6 +171,35 @@ export function annotateFrame(frame: Frame) {
   return frame;
 }
 
+function annotateBabelAsyncFrames(frames: Frame[]) {
+  const firstIndex = findIndex(
+    frames,
+    frame =>
+      getFrameUrl(frame).match(/regenerator-runtime/i) &&
+      frame.displayName === "tryCatch"
+  );
+
+  const lastIndex = findIndex(
```

Comment:
can `getFrameUrl(frame)` return null? if so, we should have a check to see if the URL is null before calling match

Reply:
The flowtype says sourceUrl is optional. Will add a null check.

Reply:
nevermind. getFrameUrl does a null check